server/protocols/jmap/src/main/java/org/apache/james/jmap/model/Message.java

```diff
@@ -130,12 +132,13 @@ private static String getPreview(IndexableMessage im) {
     }
 
     @VisibleForTesting static String computePreview(String body) {
-        if (body.length() <= 256) {
-            return body;
+        String bodyWithoutTag = Jsoup.parse(body).text();
```

Comment:
Compute preview might be done in an other class, so that preview is not coupled with message.